client/my-sites/stats/features/modules/stats-locations/stats-locations.tsx

```diff
@@ -200,10 +221,26 @@ const StatsLocations: React.FC< StatsModuleLocationsProps > = ( { query, summary
 		{ label: 'Netherlands', countryCode: 'NL', value: 500, region: '155' },
 		{ label: 'Spain', countryCode: 'ES', value: 400, region: '039' },
 	];
+
 	const hasLocationData = Array.isArray( data ) && data.length > 0;
 
 	const locationData = shouldGate ? fakeData : data;
 
+	const heroElement = (
+		<>
+			<Geochart data={ locationData } geoMode={ geoMode } skipQuery customHeight={ 480 } />
+			{ geoMode !== 'country' && ! summaryUrl && (
+				<CountryFilter
+					countries={ countriesList }
+					defaultLabel={ optionLabels[ selectedOption ].countryFilterLabel }
+					selectedCountry={ countryFilter }
+					onCountryChange={ onCountryChange }
+					tooltip={ divisionsTooltip }
+				/>
+			) }
+		</>
+	);
```

Comment:
I've extracted this element to avoid stacking multiple props/components in the inline declaration, but I can move it back if preferable.

Reply:
Sounds sensible to me! 👍